### ISSUE TITLE
opencv-python dependency issues

### ISSUE THREAD

**Issue by Selmaan** [lambdaloop/camibrate#2]
Hey, thanks for this package! I think there's a problem in the way opencv-python dependencies are listed though? 

My understanding is that one is only supposed to have a single opencv installation in an environment, but you have both opencv-python and opencv-contrib-python. See here for the error I was getting and resolution: https://stackoverflow.com/questions/45972357/python-opencv-aruco-no-module-named-cv2-aruco

Basically, shouldn't the dependency for the package just be opencv-contrib-python~=3.4, so that you don't have conflicting namespaces for cv2 and cause issues importing aruco?

**Comment by lambdaloop** [comment]
Ah that's interesting, I didn't realize that it conflicted in this way, thank you for the pointer!
I've removed the opencv-python dependency as you mentioned.
